test_main.py

```diff
@@ -1,6 +1,8 @@
-from .repositories import create_book, get_books, get_book, update_book, delete_book
+from sqlalchemy import inspect
+
```

Comment:
```suggestion
```suggestion
from . import models
```
```